### PR TITLE
Ncnews 9

### PR DESCRIPTION
**Pull request by NookaCoder**
This ticket:
Adds /api/articles/:article_id/comments endpoint
Tests for 200 response with requested array
Tests for 400 and 404 errors for bad ids

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -159,6 +159,44 @@ describe('/api/articles/:article_id', () => {
 });
 
 
+describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 responds with an array of comment objects with required properties', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
```
Comment (Poonam-raj) [inline __tests__/app.test.js:168]: should also check the length of the comments array is what you expect

File: __tests__/app.test.js [hunk 2]
```diff
@@ -159,6 +159,44 @@ describe('/api/articles/:article_id', () => {
...
+    });
+    test('GET:404 sends appropriate status and error message when given a valid but non-existent id', () => {
+        return request(app)
+            .get('/api/articles/3141/comments')
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe('Article does not exist')
+            })
+    });
+});
```
Comment (Poonam-raj) [inline __tests__/app.test.js:202]: I'd also check for a article which has no comments associated with it (should still be 200 OK)

**Review by Poonam-raj — APPROVED** [review]
All good :D few comments in your testing but nothing major so feel free to address those and merge when you're ready

File: __tests__/app.test.js (hunk 2 above)
Comment (NookaCoder) [reply __tests__/app.test.js:202]: Hey, just wondering how I test for that if all the articles have comments?

**Pull request merged**